rich/logging.py

```diff
@@ -171,7 +174,9 @@ def render_message(self, record: LogRecord, message: str) -> "ConsoleRenderable"
         if highlighter:
             message_text = highlighter(message_text)
 
-        if self.KEYWORDS:
+        if self.keywords:
+            message_text.highlight_words(self.keywords, "logging.keyword")
+        elif self.KEYWORDS:
```

Comment:
Tried to keep changes minimal, `render_message()` will first check to see if `self.keywords` is set, and fall back on `self.KEYWORDS` if it's not.